runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPanel.java

```diff
@@ -131,39 +142,105 @@ private static String htmlLabel(String key, long value)
 		return String.format(HTML_LABEL_TEMPLATE, ColorUtil.toHexColor(ColorScheme.LIGHT_GRAY_COLOR), key, valueStr);
 	}
 
-	void addLog(final String eventName, final int actorLevel, LootTrackerItemEntry[] items)
+	void addEntry(final String eventName, final int actorLevel, LootTrackerItemEntry[] items, boolean groupLoot)
+	{
+		final String subTitle = actorLevel > -1 ? "(lvl-" + actorLevel + ")" : "";
+
+		LootTrackerEntry entry = new LootTrackerEntry(eventName, subTitle, System.currentTimeMillis(), items);
+		entries.add(entry);
+
+		if (!groupLoot || !stackEntry(entry))
```

Comment:
Use list.contains. Also, this logic is very cryptic. 
I would just make the entries map, and do:
```
existingEntry = map.get(e.getname());
if (entry != null)
{
  existingEntry.combineWith(entry);
}

overallGp = map.values().stream().map(Entry::getTotalPrice()).sum();
```

also probably I would do that by using map.compute or something.